bindings/python/convert.py

```diff
@@ -65,7 +81,10 @@ def convert_single(model_id):
     )
     args = parser.parse_args()
     model_id = args.model_id
-    try:
-        convert_multi(model_id)
-    except EntryNotFoundError:
+    api = HfApi()
+    info = api.model_info(model_id)
+    filenames = set(s.rfilename for s in info.siblings)
+    if "pytorch_model.bin" in filenames:
         convert_single(model_id)
+    else:
+        convert_multi(model_id)
```

Comment:
This could also return the path to the lfile/list of files so the try/finally is just around those instructions and does not take a whole function.

Reply:
But if the function crashes and has not try except within the body, then there's no way to capture the llist of files right ?